doc/howtos/software-update-server.rst

```diff
@@ -219,26 +217,24 @@ from the command in the Example is:
     tmp-glibc/deploy/swupd/intel-corei7-64/ostro-image-swupd/
 
   This folder contains both data from intermediate steps and the actual
-  SW Update stream.
-
-  Its location is::
+  SW Update stream located here::
 
     tmp-glibc/deploy/swupd/intel-corei7-64/ostro-image-swupd/www/
 
-  This folder will contain also data related to following builds and it is
-  the one that must be exposed to the device in need of maintenance, through
-  a web server, for example nginx or apache.
+  This folder will also contain data related to subsequent builds and
+  must be exposed through a web server (e.g., ``nginx`` or ``apache``) 
+  to the device running the SW update client for getting its maintentance.
```

Comment:
The version of swupd-client we use currently hardcodes the default URL to ClearLinux's site. meta-swupd is patched to point to example.com. The latest version of swupd-client doesn't have any hardcoded URLs and reads the default one from config files `/usr/share/defaults/swupd/versionurl`, `/usr/share/defaults/swupd/contenturl`.